Monika After Story/game/script-story-events.rpy

```diff
@@ -1291,19 +1295,16 @@ label .afterdontjoke:
                 m "Nevermind, I found it."
                 window hide
 
-    # NOTE: add a sound for light switch?
-
-    # turn on the lights
-    play sound closet_open
-    call spaceroom(hide_monika=True, scene_change=True, show_emptydesk=False)
+    # turn on lights
+    play sound "mod_assets/sounds/effects/light-switch-sound-effect.mp3"
```

Comment:
I think we should define our audio files as well so it's easier to reuse that type the whole filepath to it.